utils/prometheus-node-exporter-lua/files/usr/lib/lua/prometheus-collectors/dnsmasq.lua

```diff
@@ -0,0 +1,41 @@
+local ubus = require "ubus"
+
+local function scrape_leasefile(name, leasefile)
+  metric_dhcp_lease = metric("dhcp_lease", "gauge")
+  for e in io.lines(leasefile) do
```

Comment:
uci can return something that is configured but not running, can you make sure we fail gracefully / silently if the file doesn't exists

Reply:
I tried changing the implementation not to output any metric if the file is not there or dnsmasq is not reachable from ubus.
Can you give me some feedback? I have no experience with lua.

In this way there is no feedback to prometheus if dnsmasq is not running.
I do not need it, but users might like having a conventional `dnsmasq_up` variable reporting that it's not running. What do you think?